Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/RasterPass.h

```diff
@@ -48,6 +70,8 @@ namespace AZ
         protected:
             explicit RasterPass(const PassDescriptor& descriptor);
 
+            void DeclareAttachmentsToFrameGraph(RHI::FrameGraphInterface frameGraph) const;
```

Comment:
Needs a comment describing the purpose of the method

Reply:
Done.
```
            //! Binds all attachments to the current scope in the frameGraph.
            //! Similar functionality as RenderPass::DeclareAttachmentsToFrameGraph(), but
            //! RasterPass also binds "SubpassInput" attachments where applicable. 
```